src/test/java/com/epam/tc/hw4/ex1/Exercise1.java

```diff
@@ -0,0 +1,53 @@
+package com.epam.tc.hw4.ex1;
+
+import com.epam.tc.hw4.page.BasePage;
+import com.epam.tc.hw4.page.IndexPage.IndexPage;
+import com.epam.tc.hw4.page.IndexPage.IndexPageHeader;
+import com.epam.tc.hw4.page.IndexPage.IndexPageLeftMenu;
+import io.qameta.allure.Feature;
+import io.qameta.allure.Story;
+import org.testng.annotations.Test;
+
+import static io.qameta.allure.Allure.step;
+
+public class Exercise1 extends BasePage {
+
+    @Test(description = "Test login and filling content Index Page")
+    @Feature("Authorization")
+    @Story("Login on IndexPage")
+
+    public void indexPageLoginAndFillingPageContentTest() {
+        attachProperty();
```

Comment:
do you need this method here? You already have it in your base class so has it been added for the report or?